README.rst

```diff
@@ -1,4 +1,5 @@
 Globus Compute - Fast Function Serving
+
 =============================
```

Comment:
Doh!  I think rather that the intent was to have the first line be a header.  This is ReSTructured text (`.rst`), so the header specification (`====...`) needs to be the same number of characters as the line above:
```reST
Globus Compute - Fast Function Serving
======================================
...
```

I suspect we flubbed this when we did the rebrand last April.

I'll leave the glory for you.  Go go go!